sdk/storage/azure-storage-blob-cryptography/src/test/java/com/azure/storage/blob/specialized/cryptography/EncryptedBlobClientBuilderTests.java

```diff
@@ -303,14 +304,33 @@ public void encryptedRegionLength(long regionLength) {
 
     @Test
     public void encryptedRegionLengthDefault() {
-        EncryptedBlobClient encryptedBlobClient = new EncryptedBlobClientBuilder(EncryptionVersion.V2)
+        EncryptedBlobClient encryptedBlobClient = new EncryptedBlobClientBuilder(EncryptionVersion.V2_1)
```

Comment:
For this test, we should not need to update the version as we are not setting a new authenticated region length.